components/contribution-flow/ContributionFlowHeader.js

```diff
@@ -57,7 +58,7 @@ class NewContributionFlowHeader extends React.Component {
               />
             </H1>
           </CollectiveTitleContainer>
-          {contributors?.length > 0 && (
```

Comment:
We corrected this in past PR

Reply:
@znarf : Hmm, maybe I did something wrong with the merge, let me recheck and redo this merge so that we are doubly sure. 🤔

Reply:
Yes, my mistake, I missed this in resolving merge conflicts, I've done the conflict resolution again and this time it's good I think. 👍🏽